docs/source/app_developers_guide/about_event_subscriptions.rst

```diff
@@ -0,0 +1,137 @@
+**************************
+About Events Subscriptions
+**************************
+
```

Comment:
I would add a little motivation here to answer "Why do I care what an event subscription is?" Something like "Event subscriptions are how you tell a validator what information you would like to be notified about."